docs/docs/user/examples/atm-fraud-pipeline.md

```diff
@@ -79,7 +79,6 @@ kubectl port-forward --namespace kpops service/k8kafka-cp-kafka-connect 8083:808
     ```shell
     poetry run kpops deploy ./examples/bakdata/atm-fraud-detection/pipeline.yaml \
     --pipeline-base-dir ./examples \
-    --defaults ./examples/bakdata/atm-fraud-detection/defaults \
```

Comment:
I just tried running this command without `--defaults` and this is actually broken right now. most likely because of https://github.com/bakdata/kpops/pull/138/files#r1123381257

Reply:
Reverted,

Could be once again removed in #144